### ISSUE TITLE
Invalid Image ID

### ISSUE THREAD

**Issue by charleskaminski** [tobami/overmind#6]
Steps to recreate:
1. Created a new "EC2 US East" provider.
2. Click on new node
3. Select the above provider
4. In name text box type "Test"
5. Click on "Add new image"
6. In the "Type an image id" text box type "ami-a6f504cf"
7. Click the button "Add image"

"Invalid image id" and "this field is required" errors are returned.

**Comment by tobami** [comment]
Can you find the "ami-a6f504cf" image in the combo box?
The name should be "ubuntu-images/ubuntu-maverick-10.10-i386-server-20101225"

I know it is difficult to find a name in a list with thousands of entries (that's the reason the id field is there), but it may be another hint

**Comment by charleskaminski** [comment]
The page does not behave like that...There are no list of entries.  All the drop downs are empty.  I put a screen shot here: http://www.4freeimagehost.com/show.php?i=318a9c69bf74.png  

I checked the server logs and see only HTTP 200 codes.

**Comment by tobami** [comment]
Ok, then no wonder you got "invalid image id": it was not added to the EC2 provider.

Normally, when you add a new provider, all images and sizes are cached by Overmind. It seems like this did not happen in your case. Are you using master?

**Comment by charleskaminski** [comment]
Yes.  As downloaded from https://github.com/tobami/overmind/zipball/master

**Comment by tobami** [comment]
ok, I'll look into it!

**Comment by charleskaminski** [comment]
Any Luck?

**Comment by tobami** [comment]
Ok, it may not be a bug, just a configuration problem. Overmind 0.1 was pretty easy to setup, but master now uses a job queue with RabbitMQ and celery. What probably happens is that you didn't start celery in the background, and the EC2 provider was created, but not correctly: The images/sizes, etc were not added to the provider.

To improve this situation, I have improved the README. Can you have a look and see if it helps you get Overmind properly running?

Just remove the EC2 provider and try creating it again.

**Comment by charleskaminski** [comment]
No change with the new instructions...
Would you try creating a fresh EC2 instance on Amazon, load all of the software (following the installation instructions you've written) and then give it a try yourself?  

For me I still get "Invalid image id" and "this field is required" errors.

**Comment by charleskaminski** [comment]
Closing the issue was an accident.... please see the previous post.

**Comment by tobami** [comment]
ok, will check that out starting from the beginning

**Comment by charleskaminski** [comment]
Any luck?

**Comment by tobami** [comment]
Hi, I started from the beginning myself.

I does seem to work for me. The problem is two-fold:
- When celery is not running you get no feedback that the jobs are not going to be processed
- For EC2 the list of images is huge, and it make take 20 seconds to import all

So immediately after creating an EC2 provider I see empty drop down menus. After 20 seconds going to "new node" does show all sizes, images, etc.

In the console where you run celery, did you see the import jobs completing?
